jupyterhub/files/hub/jupyterhub_config.py

```diff
@@ -357,6 +355,16 @@ def camelCaseify(s):
     if cull_max_age:
         cull_cmd.append("--max-age=%s" % cull_max_age)
 
+    cull_proto = "http"
+    if get_config("cull.ssl_enabled", False):
+        cull_cmd.append("--ssl-enabled")
+        cull_proto = "https"
+
+    base_url = c.JupyterHub.get("base_url", "/")
+    cull_cmd.append(
+        f"--url={cull_proto}://localhost:8081" + url_path_join(base_url, "hub/api")
+    )
+
```

Comment:
Thanks for considering this @thomasv314!

@thomasv314 is it correct that you have made use of both this Helm chart@master and internal_ssl, and want to make this change to support culling before internal_ssl is properly supported for the helm chart?

---

Below I'm just writing down things that pop into my mind for some transparency, I intend not to burden you with answering them but it rather a strategy for myself to keep track of thoughts while also being transparent to you as a PR author.

- What does --ssl-enabled really do?
- What does --url really do?
- Could it be times when it makes sense for --ssl-enabled to be set without having a --url with HTTPS proto?
- With internal_ssl as a official supported feature, we probably want to adjust the URL based on having internal_ssl enabled. With that in mind, how should we avoid causing issues for anyone using this in-between fix that is only relevant for what I think is those with custom internal_ssl support?
- Is this worth adding before internal_ssl? Maybe, not sure.